test/gtest/ucp/test_ucp_rma.cc

```diff
@@ -57,12 +74,30 @@ class test_ucp_rma : public test_ucp_memheap {
         request_release(status_ptr);
     }
 
+    void get_nbi_iov(size_t size, void *target_ptr, ucp_rkey_h rkey,
+                     void *expected_data, void *arg) {
+        ucp_dt_iov_t iov[UCP_MAX_IOV];
+        ucs_status_ptr_t status_ptr;
+        ucp_request_param_t param;
+
+        param.op_attr_mask = UCP_OP_ATTR_FIELD_DATATYPE;
+        param.datatype     = DATATYPE_IOV;
+
+        for (size_t iov_count = 2; iov_count <= UCP_MAX_IOV; iov_count *= 2) {
+            status_ptr = do_get_iov(size, target_ptr, rkey, expected_data,
+                                    &param, iov, iov_count);
+            flush_ep(sender());
+            request_release(status_ptr);
+        }
+    }
+
+
 protected:
     static size_t default_max_size() {
         return (100 * UCS_MBYTE) / ucs::test_time_multiplier();
     }
 
-    void test_mem_types(send_func_t send_func,
+    void test_mem_types(send_func_t send_func, size_t min_size = 128,
```

Comment:
why needed?

Reply:
Because existing tests & function calls shouldn't change (see https://github.com/openucx/ucx/pull/7317/files#diff-33e7961eaa7cabfec61921393c5704c29ab4a3a57026a203e9aa68e9fa843155L127)

Reply:
ok i guess the answer is to test with minimal size which is not always 128